raft-tech/TANF-app
# update develop and staging envs to use acf.hhs.gov domain

## Summary of Changes
The deploy scripts and environment variables have been updated to expect the acf.hhs.gov domain.
Pull request closes #2385 

## How to Test
For testing, this change is currently deployed in develop, because the domains are only set up for develop and staging. Testing includes checking full range of the site, making sure each page comes up as expected. 

1. Open https://tdp-frontend-develop.acf.hhs.gov/ and sign in.
1. Proceed with functional tests


## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] make sure site is available at https://tdp-frontend-develop.acf.hhs.gov/
+ [x] navigation from there proceeds as normal
+ [x] during navigation from the site it never redirects away from the acf.hhs.gov domain

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

This change did not include frontend or backend code changes, just pipeline and environment changes

+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
